.tekton/pull-request.yaml

```diff
@@ -130,10 +130,9 @@ spec:
               image: registry.redhat.io/openshift4/ose-cli:v4.10
               script: |
                 #!/usr/bin/env bash
-                # Perform cleanup of resources created by e2e test run
-                # TODO: Application and Component names should be unique, so the cleanup task doesn't affect other pipelineruns
-                oc delete --ignore-not-found application build-suite-test-application -n $(params.e2e_test_namespace)
-                oc delete --ignore-not-found component build-suite-test-component-git-source -n $(params.e2e_test_namespace)
+                # Perform cleanup of resources created by gitops service
+                oc delete --ignore-not-found deployment --all -n $(params.e2e_test_namespace)
+                oc delete --ignore-not-found eventlisteners --all -n $(params.e2e_test_namespace)
```

Comment:
I added this to get rid of these kinds of deployments (+ event listeners)
![Screenshot 2022-08-23 at 12 49 07](https://user-images.githubusercontent.com/4881144/186150479-1ea986fc-4236-40a5-867b-1274c37b3721.png)